src/app/Shared/Checkboxes.component.tsx

```diff
@@ -11,6 +11,7 @@ export type CheckboxesConfig = {
 export interface CheckboxesProps {
     onCheckboxChange: (name: string, checked: boolean) => void;
     checkboxes: CheckboxesConfig[];
+    style: SerializedStyles | null;

```

Comment:
Consider putting the actual styles in this component and use a flag here. E.g.: ``useSmallLabels: boolean`` or whatever the difference is in your app. The background is that your view components main task is to display things in specific way or design. If you make the style configurable by external components you loose this encapsulation and split the styling responsibility over different components. E.g. if you want to change the overall checkbox design you have to change all components using the checkboxesComponent with a specific style too.

Reply:
But the style is the same for the checkboxes and the radio buttons in the options. Therefore I thought it makes more sense to define the style there, and not in two different components (1x in options and 1x in checkboxes).

Reply:
Ok, i understand. In this case you could use a common style in a separate file that both of the shared components (radio and checkboxes) would use. You could place it to the other common app wide styles. In general it is a hard decision to how and where you integrate the css styles in your app. So for app specific components i would prefer my suggestion. If you want to write a general component used by different applications your pattern would be the better one.

Reply:
Done. "Implement shared toggle style"